test/cli/Makefile.am

```diff
@@ -36,24 +36,24 @@ test83-check:
 	$(TEST83) $(PROJEXE)
 
 testvarious-check:
-	PROJ_LIB=$(PROJ_LIB) $(TESTVARIOUS) $(CS2CSEXE)
+	PROJ_IGNORE_USER_WRITABLE_DIRECTORY=YES PROJ_LIB=$(PROJ_LIB) $(TESTVARIOUS) $(CS2CSEXE)
```

Comment:
I worry that ``PROJ_IGNORE_USER_WRITABLE_DIRECTORY`` isn't a strong enough name. You are using it negated in this incantination. Does this variable mean to not write into user directories? Not read any data from user directories? It isn't quite clear.

Reply:
changed to PROJ_SKIP_READ_USER_WRITABLE_DIRECTORY